app/models/policy_version.rb

```diff
@@ -86,6 +86,50 @@ def before_save
     self.policy_sha256 = Digest::SHA256.hexdigest(policy_text)
   end
 
+  def after_save
+    log_versions_off_limit
```

Comment:
how expensive is this log? i see that you make another call to the DB for this

Reply:
@eranha, what about changing these names to something like `log_versions_to_expire` and `remove_expired_policy_versions` for these two method names? `log_versions_off_limit` and `delete_versions_off_limit` don't give me a lot of context to what these methods might do.

Reply:
Great names